core/store/migrations/0024_create_log_config_table.go

```diff
@@ -0,0 +1,41 @@
+package migrations
+
+import (
+	"gorm.io/gorm"
+)
+
+const up24 = `
+	CREATE TYPE log_level AS ENUM (
+		'debug',
+		'info',
+		'warn',
+		'error',
+		'panic'
+	);
+	
+	CREATE TABLE log_configs (
+		"id" BIGSERIAL PRIMARY KEY,
+		"service_name" text NOT NULL UNIQUE,
+		"log_level" log_level NOT NULL,
+		"created_at" timestamp with time zone,
+		"updated_at" timestamp with time zone
+	);
+`
+
+const down24 = `
+	DROP TABLE IF EXISTS log_configs;
+
+	DROP TYPE IF EXISTS log_level;
+`
+
+func init() {
+	Migrations = append(Migrations, &Migration{
+		ID: "0024_create_log_config_table",
```

Comment:
Make sure `0024` hasn't been used yet since this PR has been open for a while

Reply:
updated ~